### DISCUSSION TITLE
What is the difference between cellID and baseCellID?

### DISCUSSION THREAD

**Discussion by mojtabaAmiraslanpour** [UvaCsl/HemoCell#32] [General discussion]
Hello all,

I am working with the CSV files right now and want to extract data for each particular PLT, but found out that there are two types of IDs corresponding to each cell, cellID, and baseCellID. So I am a little confused about it. Can you shed some light on this?

**Comment by MaxvdKolk** [comment]
There are two IDs defined to keep track of cells that cross atomic blocks and are therefore represented in two (or more) blocks (e.g. CPUs) at a time. Having both a `cellID` and `baseCellID` allows to discriminate the "original" and their "copies" and possibly filter out the duplicates, for instance in post-processing, I think.

**Comment by gzavo** [comment]
Exactly! Plus the cellID is increased by number_of_cells every time a cell trespasses a periodic boundary, so you can tell in the end how many round-trips a particular cell has done.

**Comment by mojtabaAmiraslanpour** [comment]
Thanks,
So as its name says, _baseCellID_ must be the original cell and all other _cellID_'s corresponding to this _baseCellID_ are its copies. Is that right?
I found out that when _cellID_ is positive it equals _baseCellID_ but when it is negative it doesn't equal it.
